android/app/src/main/AndroidManifest.xml

```diff
@@ -46,10 +45,10 @@
       <activity
         android:name=".MainActivity"
         android:label="@string/app_name"
-        android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode"
+        android:configChanges="keyboard|keyboardHidden|orientation|screenLayout|screenSize|smallestScreenSize|uiMode"
```

Comment:
Maybe we should check how our app behaves on split screens as it seems to be the default configuration now

Cf: https://developer.android.com/guide/topics/large-screens/multi-window-support

Reply:
Looks good Tested logging in, navigating between apps and some apps (Drive, Notes, Settings, CoachCO2, mespapiers).